cmd/configure.go

```diff
@@ -636,11 +674,16 @@ func configureGithub(ctx context.Context, _flags ConfigureGithubFlags) error {
 		actionPath = fmt.Sprintf("%s/actions", remoteURL)
 	}
 
-	agenda = append(agenda, fmt.Sprintf("• Setup a Speakeasy API Key as a GitHub Secret - %s/workspaces/%s/apikeys", core.GetServerURL(), workspaceID))
-	agenda = append(agenda, fmt.Sprintf("• In your repo navigate to %s and setup the following repository secrets:", secretPath))
+	if !autoConfigureRepoSuccess {
+		agenda = append(agenda, fmt.Sprintf("• Setup a Speakeasy API Key as a GitHub Secret - %s/workspaces/%s/apikeys", core.GetServerURL(), workspaceID))
```

Comment:
What does agenda mean in this context

Reply:
It's just the message we post at the end of configure. The instructions on what they need to do. They don't need to create an API key, because we did it for them if they were github app linked